lldb/include/lldb/Utility/Stream.h

```diff
@@ -23,6 +23,16 @@
 
 namespace lldb_private {
 
+struct Information {
```

Comment:
This should be within the Stream class.

Also the name is too generic, `HighlightSettings` maybe? Choosing a name is hard but something that says what the information is about.

Reply:
Done.

I was thinking: should we add a default constructor?

Reply:
Looks like you already did this and yes it's a good thing, it means folks can't forget to give all 3 bits of information.